apps/designer/app/designer/features/breakpoints/breakpoints.tsx

```diff
@@ -120,7 +120,7 @@ export const Breakpoints = () => {
       </DropdownMenuTrigger>
       <DropdownMenuPortal>
         <DropdownMenuContent
-          css={{ zIndex: "$1" }}
+          css={{ zIndex: variables.zIndices[1] }}
```

Comment:
Is this indexed access safe? Can you use here out of bound index?

Reply:
Good point! But yes, seems like safe https://www.typescriptlang.org/play?ts=4.9.4#code/MYewdgzgLgBFCm0YF4YG8BQNswEQEZcAuGfAGix1wCZiZqKBfGAQwhlEigG4MNOk4eCjiIoAbXwBdfuCRQAFgCd4w1AmjiAzFKA